### PR TITLE
install target

### PR DESCRIPTION
**Pull request by Thaodan**
needs to be extended when the other PRs of mine get merged.
Also you should mention that adding CXXFLAGS+="-DNV_DRIVER_PATH=<lib>"  during make should work too instead of editing the source.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,14 @@
+DESTDIR       =
```
Comment (leonmaxx) [inline Makefile]: Shouldn't it be:
DESTDIR?=
PREFIX=$(DESTDIR)/usr/local

Comment (leonmaxx) [reply Makefile]: This will still override `DESTDIR` with empty string.

Comment (Thaodan) [reply Makefile]: Fixed that.

**Comment by Thaodan** [comment]
Rebased for current master

**Pull request merged**